src/aliceVision/hdr/brackets.cpp

```diff
@@ -176,9 +178,13 @@ void selectTargetViews(std::vector<std::shared_ptr<sfmData::View>> & out_targetV
                 {
                     ALICEVISION_THROW_ERROR("File: " << lumaStatFilepath << " is not a valid file");
                 }
-                lumaMeanMean += lumaMean;
+                if (exposure > 0.0) // discard dummy luminance info (with exposure set to -1.0) added at calibration stage if samples are missing for a view
+                {
+                    lumaMeanMean += lumaMean;
+                    nbValidViews++;
```

Comment:
use preincrement by default and post only when needed

Reply:
Done